src/main/webapp/creatives/maps/simid_map.html

```diff
@@ -3,7 +3,7 @@
     <link rel="stylesheet" type="text/css" href="simid_map.css">
   </head>
   <body>
-    <div id="container">
+    <div id="iframeContainer">
```

Comment:
This is a much better name!! I definitely did not know that was originally referring to the entire iframe.

Reply:
Careful here -- this isn't the "entire" iframe .. this is just the main div in the iframe.  The "entire" iframe is the actual `<iframe></iframe>` element.  All of the code you are writting will be run from inside your iframe.  Therefore if you wanted to access the "entire" iframe, you could call `window.document`.  That would include _everything_, including the actual `<html></html>` element.  

Imo a name like "mainContainer" or "adContainer" might convey that a bit better

Reply:
okay I will change to adContainer